### PR TITLE
refactor：menu control

### PR DESCRIPTION
**Pull request by Clauszy**
refactor：menu control

### CODE REVIEW

**Review by itsXuSt — DISMISSED** [review]
+1

File: src/plugins/common/core/dfmplugin-menu/menuscene/dconfighiddenmenuscene.cpp [hunk 1]
```diff
@@ -30,13 +31,40 @@ QString dfmplugin_menu::DConfigHiddenMenuScene::name() const
     return DConfigHiddenMenuCreator::name();
 }
 
+bool DConfigHiddenMenuScene::initialize(const QVariantHash &params)
+{
+    auto currentDir = params.value(MenuParamKey::kCurrentDir).toUrl();
+    if (currentDir.isValid()) {
+        // extend menu scene
+        if (Helper::isHiddenExtMenu(currentDir))
+            diableScene();
```
Comment (Johnson-zs) [inline src/plugins/common/core/dfmplugin-menu/menuscene/dconfighiddenmenuscene.cpp]: 单词拼写

**Review by Johnson-zs — APPROVED** [review]
+1

**Pull request merged**